src/main/java/com/hartwig/platinum/kubernetes/KubernetesCluster.java

```diff
@@ -54,7 +54,11 @@ public PlatinumResult submit() {
         for (Supplier<PipelineInput> inputSupplier : pipelineInputs) {
             try {
                 PipelineInput pipelineInput = inputSupplier.get();
-                String sampleName = pipelineInput.tumor().map(SampleInput::name).orElseThrow();
+                String sampleName = pipelineInput.tumor()
+                        .map(SampleInput::name)
+                        .orElseGet(() -> pipelineInput.reference()
+                                .map(SampleInput::name)
+                                .orElseThrow(() -> new IllegalArgumentException("At least one tumor or reference sample must be provided")));
```

Comment:
Could be written a little more concisely:

```suggestion
 pipelineInput.tumor()
                        .or(pipelineInput::reference)
                        .map(SampleInput::name)
                        .orElseThrow(() -> new IllegalArgumentException("At least one tumor or reference sample must be provided"))
```